### PR TITLE
Document torch.quantile interpolation kwarg

### PR DESCRIPTION
**Pull request by anjali411**
clone of https://github.com/pytorch/pytorch/pull/59397

This PR documents the interpolation kwarg parameter added in #49267. Now that the forward compatibility period is over, we can expose this parameter.

### CODE REVIEW

File: test/backward_compatibility/check_backward_compatibility.py [hunk 1]
```diff
@@ -109,6 +109,8 @@
     ("aten::_inverse_helper", datetime.date(2021, 12, 31)),
     ("aten::softplus_backward", datetime.date(2022, 1, 31)),
     ("aten::softplus_backward.grad_input", datetime.date(2022, 1, 31)),
+    ("aten::quantile", datetime.date(2021, 9, 30)),
```
Comment (albanD) [inline test/backward_compatibility/check_backward_compatibility.py:112]: Shouldn't this date be somewhere in 2022?

Comment (anjali411) [reply test/backward_compatibility/check_backward_compatibility.py]: yup updated it

**Review by jbschlosser — APPROVED** [review]
Stamped!

**Comment by matthewfeickert** [comment]
@anjali411 Thanks for this work. :rocket: As this changes the API by adding in the `interpolation` kwarg, should we expect to see this in the next minor release (`v1.11.0`)?

**Comment by anjali411** [comment]
> @anjali411 Thanks for this work. 🚀 As this changes the API by adding in the `interpolation` kwarg, should we expect to see this in the next minor release (`v1.11.0`)?

yeah it'll be available in the upcoming release :D

**Pull request closed without merging**